velero-plugins/imagestream/s3_test.go

```diff
@@ -0,0 +1,36 @@
+package imagestream
+
+import "testing"
+
+func TestGetBucketRegion(t *testing.T) {
+	type args struct {
+		bucket string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+		  	name: "tkaovila-aug30-velero-bsl",
+				args: args{
```

Comment:
remove extra tab?
Can we pass bucket directly to tests struct instead of `args.bucket`?

Reply:
nope.

> invalid field name args.bucket in struct literalcompiler[InvalidLitField](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#InvalidLitField)

<img width="369" alt="image" src="https://github.com/openshift/openshift-velero-plugin/assets/11228024/4daeb078-f01c-4b91-b7e6-8ba0b2b447f5">

Reply:
oh no no, I meant delete args struct and pass bucket directly to test stuct (being then name, bucket, want and wantErr)